### PR TITLE
Fixing the incorrect ticket price shown on attendees page

### LINKED ISSUE #2732: Incorrect ticket price is shown on view attendees page
**Describe the bug**

 On the ticket price column, the total price for the order is shown in place of the individual ticket price.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to 'Create and publish event'
2. Click on 'tickets<<add order'
3. Place order successfully'
4. Go back to tickets<<attendees.
5. See error

**Expected behaviour**

On the ticket price column, individual ticket price should be visible.

**Screenshots**

Screenshot 2019-04-19 06:14:30

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.04
 - Browser: Chrome
 - Version : 71.0398

**Additional context**

Working on it.

### PR DESCRIPTION
**Pull request by mohitm15**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
As per requests I had changed the heading from `ticket price` to `total price` to counter this issue

#### Changes proposed in this pull request:

- Update `list.js`

Fixes #2732 

**Screenshot**
Screenshot 2019-04-20 06:47:36

### CODE REVIEW

**Comment by mohitm15** [comment]
@CosmicCoder96 Please review this

File: app/controllers/events/view/tickets/attendees/list.js [hunk 1]
```diff
@@ -13,8 +13,8 @@ export default Controller.extend({
       title        : 'Ticket Name'
     },
     {
-      propertyName : 'ticket.price',
-      title        : 'Ticket Price',
+      propertyName : 'total.price',
```
Comment (CosmicCoder96) [inline app/controllers/events/view/tickets/attendees/list.js]: what's total?

Comment (mohitm15) [reply app/controllers/events/view/tickets/attendees/list.js]: @CosmicCoder96 Since the table shows total price.

Comment (CosmicCoder96) [reply app/controllers/events/view/tickets/attendees/list.js]: @mohitm15 I know that I am asking what's the `total` in the propertyName variable.

Comment (mohitm15) [reply app/controllers/events/view/tickets/attendees/list.js]: @CosmicCoder96 I just changed the variable name

**Comment by mohitm15** [comment]
> Okay, you can't just change a variable name randomly, without realizing what's the context you are talking about.
> Also, you are solving this the wrong way, the ideal solution would be to match attendee to the particular ticket he holds, not this.

@CosmicCoder96 I suppose to solve it in the latter way but @uds5501 said to change just the heading and mentioned you.

**Comment by uds5501** [comment]
@mohitm15 True, I suggested it, but I think @CosmicCoder96 is right

**Comment by mohitm15** [comment]
no problem @uds5501 I will make changes soon according to @CosmicCoder96

**Comment by mohitm15** [comment]
@kushthedude push PR by tomorrow

**Comment by kushthedude** [comment]
@mohitm15 Are you telling me to push pr or you will push commits 😅

**Comment by CosmicCoder96** [comment]
Inactive.

**Pull request closed without merging**